user/niri/template.py

```diff
@@ -27,24 +28,28 @@
     "udev-devel",
 ]
 depends = [
+    "cmd:bash!bash",
     "so:libEGL.so.1!libegl",
 ]
 pkgdesc = "Scrollable-tiling wayland compositor"
-maintainer = "Orphaned <orphaned@chimera-linux.org>"
+maintainer = "Erica Z <zerica@callcc.eu>"
 license = "GPL-3.0-or-later"
 url = "https://github.com/YaLTeR/niri"
 source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
-sha256 = "d8854830436a87215b0bc6a60b6d43f350d927a03a2798c75f0fbda228bac8d3"
+sha256 = "86b89bcfc3fc6a8ed81f9e3f0ac7a29bd30267515efb2c19e1e0bc2ccd67b649"
+# upstream recommendation
+nostrip_files = ["usr/bin/niri"]
```

Comment:
ironic for the infinite strip window manager

(why is that? i don't immediately see this anywhere)

Reply:
it makes the size go from like 14M to >100M and the only benefit is "don't have to install niri-dbg"